components/wifi/esp8266-driver/ESP8266/ESP8266.h

```diff
@@ -167,6 +177,14 @@ class ESP8266 {
     */
     bool disconnect(void);
 
+    /**
+    * Enable or disconnect Remote IP and Port with +IPD
```

Comment:
I guess this only enabled or disatble remote IP and port *printing* ? We can still connect to remote IPs if we  set this to false, but they address and port will not be printed?

I also think that the name of this function is misleading. Could we make it `ip_info_print` or something similar?

Reply:
Done